src/muya/lib/contentState/updateCtrl.js

```diff
@@ -17,14 +17,66 @@ const INLINE_UPDATE_FRAGMENTS = [
 const INLINE_UPDATE_REG = new RegExp(INLINE_UPDATE_FRAGMENTS.join('|'), 'i')
 
 const updateCtrl = ContentState => {
-  // handle task list item checkbox click
-  ContentState.prototype.listItemCheckBoxClick = function (checkbox) {
-    const { checked, id } = checkbox
-    const block = this.getBlock(id)
+  ContentState.prototype.setCheckBoxState = function (checkbox, checked) {
+    checkbox.checked = checked
+    const block = this.getBlock(checkbox.id)
     block.checked = checked
     checkbox.classList.toggle(CLASS_OR_ID.AG_CHECKBOX_CHECKED)
   }
 
+  ContentState.prototype.getParentCheckBox = function (checkbox) {
```

Comment:
I prefer write `getParentCheckBox ` as a util function, and put it at the top (bellow import statements) of this file, because you did not use `this` keyword in `getParentCheckBox `.

Reply:
Yes, your right.